internal/transport/http2_client.go

```diff
@@ -454,6 +456,11 @@ func newHTTP2Client(connectCtx, ctx context.Context, addr resolver.Address, opts
 	return t, nil
 }
 
+// Address return the resolver address meta info
```

Comment:
We don't need this method anymore, right? Could you please remove it from here and from the `ClientTransport` interface.